chapters/03-funcoes.md

```diff
@@ -3,11 +3,11 @@
 > “As pessoas pensam que Ciência da Computação é a arte de gênios. Na realidade é o oposto, são várias pessoas fazendo coisas que dependem uma das outras, como um muro de pequenas pedras.”
 > — Donald Knuth
 
-Você já viu valores de funções como `alert`, e como invocá-las. Funções são essenciais na programação JavaScript. O conceito de encapsular uma parte do programa em um valor tem vários usos. É uma ferramenta usada para estruturar aplicações de larga escala, reduzir repetição de código, associar nomes a subprogramas e isolar esses subprogramas uns dos outros.
+Você já viu valores de funções como `alert`, e como invocá-las. Funções são essenciais na programação JavaScript. O conceito de encapsular uma parte do programa em um valor é muito usado. É uma ferramenta usada para estruturar aplicações de larga escala, reduzir repetição de código, associar nomes a subprogramas e isolar esses subprogramas uns dos outros.
```

Comment:
:-1: O trecho original: `The concept of wrapping a piece of program in a value has many uses.`

O alteração proposta passa a ideia de que "encapsular partes do programa" é usada dando uma ideia de  **frequência**, o que de fato, se analisarmos o trecho vemos que ele se refere a aplicações no sentido de **casos de uso**, como por exemplo "reduzir repetição", "estruturar aplicações de larga escala" etc...